recipes/rasam/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "rasam" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/JoelJaeschke/rasam/archive/v{{ version }}.tar.gz
+  sha256: 68e0b45ee2f78ac3a9397f35f1259f1fe25860470aed68906589d3430db59a38
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - cmake
+    - {{ compiler('cxx') }}
+    - libgdal >=3.7.0
```

Comment:
`libgdal` is only needed under `host`.
It shouldn't be under `build`.

And you don't have to set it under `run` as the gdal recipe is defining [run_exports](https://github.com/conda-forge/gdal-feedstock/blob/e80378815e7683f0288880f5862bb689a961a9d8/recipe/meta.yaml#L75): having it in host will automatically add it as run dependency

Reply:
Sorry for the late reply, I knew I missed something. Implemented your changes now